blueprints/ember-page-title/index.js

```diff
@@ -1,3 +1,42 @@
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+
 module.exports = {
-  normalizeEntityName: function () {}
+  normalizeEntityName() {},
+
+  afterInstall(opts) {
+    let project = opts.project;
+
+    let isFastBootPresent = 'ember-cli-fastboot' in project.dependencies();
+
+    if (isFastBootPresent) {
+      let isAddon = 'ember-addon' in project.pkg;
+
+      let indexHtmlPath = isAddon ?
+        path.join(project.root, 'tests', 'dummy', 'app', 'index.html') :
+        path.join(project.root, 'app', 'index.html');
+
+      if (fs.existsSync(indexHtmlPath)) {
+        const contents = fs.readFileSync(
+          indexHtmlPath,
+          {
+            encoding: 'utf8'
+          }
+        );
+
+        const titleMatches = contents.match(/<title>(.*)<\/title>/i);
```

Comment:
While this works, maybe there are some well formed ways to do edits like this from blueprints?

Reply:
I'm not aware of any, but @rwjblue might be more familiar.

Reply:
In theory, we could use ember-template-recast here but that has issues with `<DOCTYPE`.

I think (since `<title>` doesn't allow nesting, and the location is stable/knowable) that this is fine...